packages/ui/src/components/RoundButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button, ButtonProps, Circle } from "@chakra-ui/react"
+
+export const RoundButton = (props: ButtonProps) => {
+  return (
+    <Button {...props}>
```

Comment:
Looks like could remove the inner `Circle` component and keep default bg and hover behaviour for consistency

```suggestion
    <Button {...props} rounded="full" size="auto" p="0.5em">
```